packages/ckeditor5-remove-format/tests/integration.js

```diff
@@ -65,6 +66,19 @@ describe( 'RemoveFormat', () => {
 					'<imageBlock src="assets/sample.png"><caption>caption</caption></imageBlock><paragraph>bar]</paragraph>' );
 		} );
 
+		it( 'does not remove image width attribute', () => {
+			setModelData( model, '<paragraph>foo[<$text bold="true">foo</$text></paragraph>' +
```

Comment:
```suggestion
			// (#9684)
			setModelData( model, '<paragraph>foo[<$text bold="true">foo</$text></paragraph>' +
```

It's a good idea to reference the issue that made to make this decision.